controllers/resource/resource_generator.go

```diff
@@ -164,6 +179,59 @@ func (r *resourceGenerator) GetPodTemplate(doguResource *k8sv1.Dogu, dogu *core.
 	return podTemplate, nil
 }
 
+func getChownInitContainer(dogu *core.Dogu, doguResource *k8sv1.Dogu) (*corev1.Container, error) {
+	// Skip chown volumes with dogu-operator client because these are volumes from configmaps and read only.
```

Comment:
I think this comment already exists inside of `filterVolumesWithClient()`. Why the duplication?